docs/source/api/apollo-server.md

```diff
@@ -350,3 +350,13 @@ addMockFunctionsToSchema({
 *  `maskErrorDetails`: boolean
 
    Set to true to remove error details from the traces sent to Apollo's servers. Defaults to false.
+
+* generateClientInfo?: (o: { context: any, extensions?: Record<string, any>}) => ClientInfo;
+
+   Creates the client information that is attached to the traces sent to the
+   Apollo backend. The context field is the execution context passed to the
+   resolvers and the extensions field is the field provided in the request's
+   query, operationName, and variables. `ClientInfo` contains fields for
+   `clientName` and `clientVersion`, which can both be optional. The default
+   value copies the respective fields from the `extensions`'s `clientInfo`
```

Comment:
Is it clear enough to just say "the `extensions`'s field" here, especially since query extensions are not standard? Maybe we should say the POST body's `extensions.clientInfo` field?